spec/helpers/users_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the UsersHelper. For example:
+#
+# describe UsersHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```

Comment:
* I get an error here when i run `rspec` kindly fix this error  HINT 💭 I tried commenting out lines 13 to 15 and got some test passing but 3tests  were still failing below is the screenshot of the new error i got after modifying this file 
![image](https://user-images.githubusercontent.com/52497246/186381731-08ad943d-0f85-4551-ad7d-977e8a0b7efd.png)
## new error 
![image](https://user-images.githubusercontent.com/52497246/186383103-c7419df1-20ba-4a3f-bdaa-06b887408f40.png)

Reply:
@Davidosky007 This file is not a part pf the tests

Reply:
I know that but if any visitor of your repo decides to run your test they will definitely have that error

Reply:
Okay. Thank you for explaining @Davidosky007